src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -446,6 +446,54 @@ public async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestP
                 repoName,
                 id);
         }
+        public async Task<IList<Commit>> GetPullRequestCommitsAsync(string pullRequestUrl)
+        {
+            (string accountName, string projectName, string repoName, int id) = ParsePullRequestUri(pullRequestUrl);
+            VssConnection connection = CreateVssConnection(accountName);
+            GitHttpClient client = await connection.GetClientAsync<GitHttpClient>();
+            var prInfo = await client.GetPullRequestAsync(repoName, id, includeCommits: true);
+            IList<Commit> commits = new List<Commit>(prInfo.Commits.Length);
+            for (int i=0; i < prInfo.Commits.Length; i++)
+            {
+                commits.Add(new Commit(prInfo.Commits[i].Author.Name == "DotNet-Bot"? "dotnet-maestro[bot]" : prInfo.Commits[i].Author.Name, 
+                    prInfo.Commits[i].CommitId ,
+                    prInfo.Commits[i].Comment));
+            }
+            return commits;
+        }
+
+        /// <summary>
```

Comment:
This isn't a helpful comment... it's mostly just the method name and parameters with spaces.  I'd delete the comment (or fill it out so it's more interesting).

Naming it "merge" is odd... is it really merging something? I would have expected that to be a call to client.Merge, not update.

Reply:
This actually merges the PR specifically dependency update pr. The API call is weird, we have to use updatePullRequest to actually merge the PR. I mentioned in the summary that it merges a dependency update pull request.

Reply:
I guess calling it Merge is fine, but the comment still isn't really adding any value.  Its just the name of the method/parameters with spaces instead of capital letters.

In my opinion, comments should provide some tangible value, or they shouldn't exist.  So either this comment needs a lot more content to that it's useful, or we should just remove the comment and let the method name/parameters speak for themselves.

Reply:
I think this method is self explanatory. I will just remove it.